deployments/webhook.yaml

```diff
@@ -14,6 +14,12 @@ webhooks:
         namespace: ${NAMESPACE}
         path: "/mutate"
       caBundle: ${CA_BUNDLE}
+    namespaceSelector:
```

Comment:
unrelated comment to this change:

@zshi-redhat i see that in sriov-network-operator we are deploying this with `failurePolicy: Ignore`
which is why i think we never faced the issue when deploying with the operator.

should we align here (separate PR ofc) ?

Reply:
@adrianchiris In sriov-network-operator, it sets the `failurePolicy` to `Ignore` upon sriov-operator pod exist and reset (automatically by operator reconcilation) `failurePolicy` to `Fail` when sriov-operator pod is back/reinstalled.

Reply:
Code change in sriov-network-operator: https://github.com/k8snetworkplumbingwg/sriov-network-operator/pull/177